scripts/core/program.yml

```diff
@@ -287,7 +287,7 @@ analogue:
     - "**clGetDeviceFunctionPointerINTEL**"
 details:
     - "Retrieves a pointer to the functions with the given name and defined in the given program."
-    - "$X_RESULT_ERROR_INVALID_FUNCTION_NAME is returned if the function can not be obtained."
+    - "RESULT_ERROR_INVALID_FUNCTION_NAME is returned if the function can not be obtained."
```

Comment:
Where is `RESULT_ERROR_INVALID_FUNCTION_NAME` defined now that `ERROR_INVALID_FUNCTION_NAME` is removed from `common.yml`?

Reply:
I would say I forgot to update this but evidently I did update it.. wrongly